### DISCUSSION TITLE
Suggested filtering criteria

### DISCUSSION THREAD

**Discussion by zanglikun** [ProxymanApp/Proxyman#2009] [General]
The filtering criteria need to support multiple criteria, similar to Chrome browser - Network settings separated by spaces, to filter URL parameters.

**Comment by NghiaTranUIT** [comment]
It's supported. You can simply hold the Command Key and click on a different filter button 👍 

<img width="693" alt="Screenshot 2024-08-08 at 8 20 24 AM" src="https://github.com/user-attachments/assets/2cda356a-24ff-42e7-8ec9-cf3faa128fcc">
